gramps/plugins/textreport/detdescendantreport.py

```diff
@@ -927,13 +928,13 @@ def write_person_info(self, person):
                 self.doc.start_paragraph('DDR-MoreDetails')
 
                 text = utils.get_address_str(addr)
+                self.doc.write_text(addr.get_schema()["title"] + ': ')
```

Comment:
As previous comment.

Reply:
Change it too:
`
                self.doc.write_text(_('{address}: ').format(address=addr.get_schema()["title"]))
`